specs/source.go

```diff
@@ -128,3 +128,11 @@ func (s *Source) Validate() error {
 	}
 	return nil
 }
+
+func (s Source) String() string {
+	pathParts := strings.Split(s.Path, "/")
```

Comment:
We can drop this bit for simplicity too (but we'll get `aws (aws@v1.0.0)` sometimes).
I'm also removing the org part of the path to make this shorter

Reply:
Same here, should check for `len(pathParts)` before going for `pathParts[1]` if it's not verified elsewhere beforehand